src/services/videoconference/index.js

```diff
@@ -448,16 +449,14 @@ class CreateVideoconferenceService extends VideoconferenceBaseService {
 			const hasStartPermission = userPermissionsInScope.includes(PERMISSIONS.START_MEETING);
 
 			if (hasStartPermission) {
-				videoconferenceMetadata = (await CreateVideoconferenceService
-					.updateAndGetVideoconferenceMetadata(scopeName, scopeId, options)).toObject();
+				videoconferenceMetadata = (await updateAndGetVideoconferenceMetadata(scopeName, scopeId, options)).toObject();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 126. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5401970)